### PR TITLE
Cleanup old docker images

### PR DESCRIPTION
**Pull request by gdbelvin**
Delete docker images older than 30 days

### CODE REVIEW

File: scripts/gcrgc.sh [hunk 1]
```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Copyright © 2017 Google Inc.
```
Comment (taknira) [inline scripts/gcrgc.sh]: It's not 2017 anymore :)

File: scripts/gcrgc.sh [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+# limitations under the License.
+#
+# Souce: https://gist.github.com/ahmetb/7ce6d741bd5baa194a3fac6b1fec8bb7
+
+IFS=$'\n\t'
+set -eou pipefail
+
+if [[ "$#" -ne 2 || "${1}" == '-h' || "${1}" == '--help' ]]; then
+  cat >&2 <<"EOF"
+gcrgc.sh cleans up tagged or untagged images pushed before specified date
```
Comment (taknira) [inline scripts/gcrgc.sh]: Out of interest, does 'clean up' mean delete?

Comment (gdbelvin) [reply scripts/gcrgc.sh]: yes

File: scripts/gcrgc.sh (hunk 1 above)
Comment (gdbelvin) [reply scripts/gcrgc.sh]: That's from the original file

**Pull request merged**